Simplified/NYPLBookCoverRegistry.m

```diff
@@ -85,6 +85,29 @@ - (NSURL *)URLForPinnedThumbnailImageOfBookIdentifier:(NSString *const)bookIdent
           URLByAppendingPathComponent:[bookIdentifier SHA256]];
 }
 
+- (void)coverImageForBook:(NYPLBook *)book handler:(void (^)(UIImage *image))handler
+{
+
+  //Thumbnail first as placeholder
+  [self thumbnailImageForBook:book handler:handler];
+
+  [[self.session
+    dataTaskWithRequest:[NSURLRequest requestWithURL:book.imageURL]
+    completionHandler:^(NSData *const data,
+                        __attribute__((unused)) NSURLResponse *response,
+                        __attribute__((unused)) NSError *error) {
+      [[NSOperationQueue mainQueue] addOperationWithBlock:^{
```

Comment:
If I'm offline then I want to fall back on either the pinned thumbnail OR the generated cover, which comes from the thumbnail handler.

I think for a time I tried to use a single method for thumbnail or cover image, adding a parameter to switch on the logic.. but made a decision to go this way with it and keep that original method untouched (probably for the fact that I wasn't interested in caching/pinning the higher res cover images).